apps/ima/translations/src/en/validation.json

```diff
@@ -1,5 +1,16 @@
 {
   "who-are-you": {
     "required": "You must select an option"
+  },
+  "arrived-date": {
+    "required": "Please provide a valid date of when you arrived or entered the UK",
+    "date": "Please enter a valid date",
+    "after": "Enter a date after 01 01 1900",
+    "before": "Please enter a date in the past"
+  },
+  "arrival-details": {
+    "required": "You must provide details about when you arrived",
+    "regex": "Arrival details must be 15,000 characters or less and must not include [ ] < > / |",
+    "notUrl": "Enter a valid explanation"
```

Comment:
I think "Please do not enter a link/url into your answers" better explains the validation

Reply:
Updated the 'notURL' validation message as suggested.